### PR TITLE
Added an appbar in activity_maps.xml

### PR DESCRIPTION
**Pull request by SyjermelAsuncion**

### CODE REVIEW

File: code/app/src/main/java/com/example/emojibrite/AttendeesActivity.java
Comment (AivanD) [inline code/app/src/main/java/com/example/emojibrite/AttendeesActivity.java:1]: what is this for?

File: code/app/src/main/java/com/example/emojibrite/ImageUpload.java
Comment (AivanD) [inline code/app/src/main/java/com/example/emojibrite/ImageUpload.java:1]: what is the difference between this ImageUpload.java and ImageUploader.java?

File: code/app/src/test/java/com/example/emojibrite/loginTest.java
Comment (AivanD) [inline code/app/src/test/java/com/example/emojibrite/loginTest.java:1]: what is this for?

File: code/app/src/main/java/com/example/emojibrite/AttendeesActivity.java
Comment (akila-edirisinghe) [reply code/app/src/main/java/com/example/emojibrite/AttendeesActivity.java:1]: this is possibly snehal's part?

**Pull request merged**